core/src/main/java/io/questdb/cairo/mig/Mig607.java

```diff
@@ -132,7 +132,7 @@ public static void migrate(
      */
     public static long readColumnTop(FilesFacade ff, Path path, CharSequence name, int plen, boolean failIfCouldNotRead) {
         try {
-            if (ff.exists(topFile(path.chop$(), name))) {
+            if (ff.exists(topFile(path, name))) {
```

Comment:
this is a benign change - in this case `chop$` wasn't necessary, but in all honesty was this change really necessary?

every time path manipulation is changed i have to trace the code to understand where \0 is. Migration was working, net effect of this change is 0 and it makes review more difficult